src/main/java/decodex/storage/Storage.java

```diff
@@ -1,5 +1,135 @@
 package decodex.storage;
 
+import decodex.ui.messages.ErrorMessages;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
 public class Storage {
 
+    /**
+     * Variables for loading from the default input filepath.
+     */
+    public static final String DEFAULT_INPUT_DIRECTORY = "input";
+    public static final String DEFAULT_INPUT_FILE = "input.txt";
+
+    /**
+     * Variable for the default output directory.
+     */
+    public static final String DEFAULT_OUTPUT_DIRECTORY = "output";
+
+    /**
+     * Initializes a new Storage.
+     */
+    public Storage() throws IOException {
+        instantiateDirectories();
+        instantiateInputFile();
+    }

```

Comment:
Perhaps only instantiate the folders but no need to instantiate the input file yet?